web/public/javascripts/controllers/datasets.js

```diff
@@ -451,21 +469,53 @@ App.DatasetController = Ember.Controller.extend({
             _this.set('currentInstance', instance.dbId);
             _this.refreshVersions(instance.dbId);
         },
+
         /**
          * Requests the security specification for the current dataset id
          * and sets the result on the controller `securitySpec`
          * @returns {Promise.<*>}
          */
-        getSecuritySpec() {
-            const securitySpecUrl = `api/v1/datasets/${this.get('datasetId')}/security`;
-            Ember.$.getJSON(securitySpecUrl, ({return_code, securitySpec}) => {
-                if (return_code === 200 && typeof securitySpec === 'object') {
-                    this.set('securitySpec', securitySpec);
-                }
+        resetSecuritySpecification() {
+            return Ember.$.getJSON(this.getUrlFor('security'), ({return_code, securitySpecification}) => {
+                return_code === 200 && this.setProperties({
+                    'securitySpecification': securitySpecification,
```

Comment:
Since you are using ES6 you could rewrite it like this:

`this.setProperties({
  securitySpecification,
  isNewSecuritySpecification: false
});`